futures-util/src/macros/poll.rs

```diff
@@ -14,25 +14,6 @@ macro_rules! try_ready {
     }
 }
 
-
-/// Extracts `Poll<T>` from `Poll<Result<T, E>>`.
-///
-/// This macro bakes in propagation of `Err` signals by returning early.
-/// This macro bakes in propagation of `Pending` and `Err` signals by returning early.
-#[macro_export]
-macro_rules! try_poll {
```

Comment:
@Matthias247 Why was this macro removed?

Reply:
@jsancio I am not aware having actively removed this in my change. Maybe it was part of @cramertj's cleanup effort, or a rebase/merge artifact. @cramertj Do you know? Otherwise it might have been a mistake.

Reply:
In https://github.com/rust-lang-nursery/futures-rs/commit/1339c878bb9ae5aeca6a6b76d54738fe35c9e587#diff-01958fc79efbf83fe4d83d5f9cb5b5ec, @cramertj  replaced uses of `try_poll!` with `?`

Reply:
Great. Thanks for the response.

Reply:
Yeah, `try_poll` is kinda useless now that `?` works on `Poll`.